flow-decomposition/src/main/java/com/powsybl/flow_decomposition/FlowDecompositionObserver.java

```diff
@@ -74,52 +76,17 @@ public interface FlowDecompositionObserver {
     void computedPsdfMatrix(Map<String, Map<String, Double>> psdfMatrix);
 
     /**
-     * Called when the AC nodal injections matrix is computed (for base case or contingency)
+     * Called after an AC loadflow is launched and terminated
      *
-     * @param positions the positions after AC loadflow
+     * @param network the network after AC loadflow
      * @param fallbackHasBeenActivated true if AC loadflow didn't converge
      */
-    void computedAcNodalInjections(Map<String, Double> positions, boolean fallbackHasBeenActivated);
+    void computedAcLoadFlowResults(Network network, boolean fallbackHasBeenActivated);
```

Comment:
Could you add an parameter "LoadFlowResult" ?
```suggestion
    void computedAcLoadFlowResults(Network network, LoadFlowResult loadFlowResult, boolean fallbackHasBeenActivated);
```